.github/workflows/publish.yml

```
@@ -0,0 +1,88 @@
+name: Publish
+on:
+  push:
+    tags:
+    - v[0-9]+.[0-9]+
+
+permissions: read-all
+
+jobs:
+  github-release:
+    name: GitHub Release
+    runs-on: ubuntu-22.04
+    steps:
```

This job requires additional permissions in order to be able to create GitHub releases.

```suggestion
  github-release:
    name: GitHub Release
    runs-on: ubuntu-22.04
    permissions:
      contents: write # To create a GitHub Release
    steps:
```